lib/string.gi

```diff
@@ -1210,6 +1210,9 @@ end);
 
 InstallGlobalFunction(StringOfMemoryAmount, function(m)
     local  whole, frac, shift, s, units;
+    if not IsInt(m) or m < 0 then
+        Error("StringOfMemoryAmount: amount must be a non-negative integer number of bytes");
+    fi;
```

Comment:
Can this still happen?

Reply:
This is a GAP callable function. Someone could pass something stupid

Reply:
Right, sorry, I meant to put this in another spot :)